odxtools/cli/_parser_utils.py

```diff
@@ -2,11 +2,22 @@
 #
 # SPDX-License-Identifier: MIT
 import argparse
+from typing import Any, Protocol, TypeVar, runtime_checkable
 
 from ..database import Database
 from ..load_file import load_file as _load_file
 
 
+@runtime_checkable
+class SubparsersList(Protocol):
+
```

Comment:
upper P in parser?

Reply:
this protocol ought to reproduce the private class `argparse._SubparsersAction`. I added a comment, thanks for the suggestion.